homeassistant/components/manual/alarm_control_panel.py

```diff
@@ -265,17 +264,34 @@ def _arming_time(self, state):
         """Get the arming time."""
         return self._arming_time_by_state[state]
 
-    def _pending_time(self, state):
+    def _delay_time(self, state):
         """Get the pending time."""
         return self._delay_time_by_state[self._previous_state]
 
     def _within_arming_time(self, state):
         """Get if the action is in the arming time window."""
         return self._state_ts + self._arming_time(state) > dt_util.utcnow()
 
+    def _get_pending_time_ends(self, state):
+        """Get the pending time window for this state and override if needed."""
```

Comment:
Why change the previous function but still call this pending?

Reply:
I will comment on both question you raised here. For me the delay is the delta. The one from the configuration or the new param coming from the trigger service. pending time is the actual clock time the state will change and there for more related to the name of the state.
I will fix the comment to get it aligned with the function name.
If you think I should do bigger change, let me know. I'm willing to do that but I do think there is a difference between the delay (delta) and pending time (clock).

Reply:
HI.  Sorry for the delay in responding (no pun intended).
As the actual configuration parameter is `delay_time` this change makes sense.